Buildings/Controls/OBC/Utilities/PIDWithAutotuning/SystemIdentification/FirstOrderTimeDelay/ControlProcessModel.mo

```diff
@@ -110,9 +110,6 @@ protected
     final k=-1)
     "Product of the normalized time delay and -1"
     annotation (Placement(transformation(extent={{-80,-70},{-60,-50}})));
-  Buildings.Controls.OBC.CDL.Reals.Abs abs1
```

Comment:
Why did we have the `abs1` instance previously, and now delete it? Are we fixing the mistake here?

Reply:
Sorry for the confusion. I deleted this because we have an <code>abs</code> instance in the <code>timConDel</code> block already.